index.js

```diff
@@ -136,6 +136,27 @@ exports.array = function(val) {
 };
 
 /**
+ * Check if an array contains only items from another array
+ * @param  {Array} arrayOfAcceptedValues
+ * @return {Function}
+ */
+exports.arrayContaining = function(arrayOfAcceptedValues) {
```

Comment:
Would something about being a subset be any clearer? `.isSubsetOf()`?

@lauriejones @ramesius